### ISSUE TITLE
I want to be able to delete my plunks from the dashboard.

### ISSUE THREAD

**Issue by rapPayne** [plnkr/feedback#415]

**Comment by mjfneto** [comment]
@rapPayne Please, check #470

**Comment by rapPayne** [comment]
@mjfneto You are correct, thanks. But it is a lot of work to delete multiples. I have >100 plunks. It would be nice (as well as more intuitive) to delete from the dashboard, IMHO. Thanks @ggoodman, for marking it as an enhancement.
